app/jobs/perform_transfer_job.rb

```diff
@@ -44,9 +44,11 @@ def perform(pending_transfer_id)
     # Our first attempt may not work if the storage provider
     previously_attempted_stored_paths = []
 
-    # This is the stored key we would ideally like to use,
+    # This is the stored path we would ideally like to use,
```

Comment:
Is this actually a path? I was under the impression that it had already been provisionally translated (un-prefixed for the bucket)?

Reply:
The reason I'm favoring path here is because the value ends up in stored_object.path.  So if we're treating ATC as storage provider agnostic, then it's not really a "key" as much as it is a "path", I think?